packages/framework/src/Form/Admin/Product/ProductFormType.php

```diff
@@ -246,176 +507,247 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'choices' => [
                     t('Automatically') => Product::PRICE_CALCULATION_TYPE_AUTO,
                     t('Manually') => Product::PRICE_CALCULATION_TYPE_MANUAL,
                 ],
+                'disabled' => $this->isProductMainVariant($product),
+                'label' => t('Sale prices calculation'),
             ])
-            ->add('orderingPriority', IntegerType::class, [
+            ->add('vat', ChoiceType::class, [
                 'required' => true,
+                'choices' => $this->vatFacade->getAllIncludingMarkedForDeletion(),
+                'choice_label' => 'name',
+                'choice_value' => 'id',
                 'constraints' => [
-                    new Constraints\NotBlank(['message' => 'Please enter sorting priority']),
+                    new Constraints\NotBlank(['message' => 'Please enter VAT rate']),
                 ],
-            ])
-            ->add('categoriesByDomainId', MultidomainType::class, [
-                'required' => false,
-                'entry_type' => CategoriesType::class,
-                'options_by_domain_id' => $categoriesOptionsByDomainId,
-            ])
+                'disabled' => $this->isProductMainVariant($product),
+                'label' => t('VAT'),
+            ]);
+
+        $productCalculatedPricesGroup = $builder->create('productCalculatedPricesGroup', ProductCalculatedPricesType::class, [
+            'product' => $product,
+            'inherit_data' => true,
+            'render_form_row' => false,
+        ]);
+
+        $builderPricesGroup->add($productCalculatedPricesGroup);
+        $productCalculatedPricesGroup
+            ->add('price', MoneyType::class, [
+                'currency' => false,
+                'scale' => 6,
+                'required' => true,
+                'invalid_message' => 'Please enter price in correct format (positive number with decimal separator)',
+                'constraints' => [
+                    new Constraints\NotBlank([
+                        'message' => 'Please enter price',
+                        'groups' => self::VALIDATION_GROUP_AUTO_PRICE_CALCULATION,
+                    ]),
+                    new Constraints\GreaterThanOrEqual([
+                        'value' => 0,
+                        'message' => 'Price must be greater or equal to {{ compared_value }}',
+                        'groups' => self::VALIDATION_GROUP_AUTO_PRICE_CALCULATION,
+                    ]),
+                ],
+                'disabled' => $this->isProductMainVariant($product),
+                'label' => t('Base price'),
+            ]);
+
+        return $builderPricesGroup;
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @param null|Product $product
+     * @return \Symfony\Component\Form\FormBuilderInterface
+     */
+    private function createSeoFormGroup(FormBuilderInterface $builder, ?Product $product)
```

Comment:
the name is inconsistent with other methods "create*Group" - you should omit the "Form" -> `createSeoGroup()` :slightly_smiling_face: